### ISSUE TITLE
Memory leak when using podman

### ISSUE THREAD

**Issue by sjC5qiX** [NilsIrl/dockerc#16]
The binary release 0.2.1 seems to keep spawning new processes, filling up memory until the OOM killer gets it when used in combination with podman.

Command used
```shell
./dockerc --image containers-storage:alpine --output alpine
```

Gif demonstrating should be attached. This happens on arch (6.6.25-1-lts) and ubuntu 22.04 with every podman image i tried.

I wasnt able to compile from source yet but if thats necessary lmk.

**Comment by Utopiah** [comment]
Indeed, same problem for me with dockerc_x86-64  v0.3.2 on podman version 4.3.1
